test/signs/utilities/signs_config_test.exs

```diff
@@ -12,10 +12,4 @@ defmodule Signs.Utilities.SignsConfigTest do
       assert Enum.member?(Signs.Utilities.SignsConfig.all_train_stop_ids(), "70056")
     end
   end
-
-  describe "all_bus_stop_ids" do
```

Comment:
Would it be useful to just temporarily comment this test out and leave a to-do to add the test back so we don't forget about it?

Reply:
I'm not convinced of the usefulness of the other tests in this file, so I don't think we need to replicate them for the bus signs. I put this test case in there early on as a safeguard to ensure we didn't accidentally turn things on too early, but now that we're ready to deploy I think it's better to just remove it.